test/acceptance/features/bindAppToService.feature

```diff
@@ -217,3 +217,56 @@ Feature: Bind an application to a service
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding Request "binding-request-empty-app" should be changed to "False"
         And jq ".status.conditions[] | select(.type=="InjectionReady").reason" of Service Binding Request "binding-request-empty-app" should be changed to "EmptyApplicationSelector"
 
+    Scenario: Bind backend service with application using bindable information specifed in CSV
+        Given Backend service "some-backend-service.v0.1.0" CSV is installed
+        * The CRD "backservs.service.example.com" is present
+            """
+            apiVersion: apiextensions.k8s.io/v1beta1
+            kind: CustomResourceDefinition
+            metadata:
+                name: backservs.service.example.com
+            spec:
+                group: service.example.com
+                versions:
+                    - name: v1
+                      served: true
+                      storage: true
+                scope: Namespaced
+                names:
+                    plural: backservs
+                    singular: backserv
+                    kind: Backserv
+                    shortNames:
+                    - bs
+            """
+        * The application CR "csv-demo-backend" is present
+            """
+            apiVersion: service.example.com/v1
+            kind: Backserv
+            metadata:
+                name: csv-demo-backend
+            spec:
+                svcName: csv-demo-backend
+            """
+        * Imported Nodejs application "csv-demo-app" is running
```

Comment:
why do we install nodejs application if we do not check later on if it really works?

Reply:
We're asserting if its deployment contains the injected secretRef or not.

Reply:
yes, but saying that application is running mean that it runs correctly, and this is not the case since we do not connect it to any database.

Reply:
Removed application.